src/types/Crediting.jl

```diff
@@ -125,8 +126,12 @@ CreditByBenchmark(;gen_col = :emis_co2e, benchmark) = CreditByBenchmark(gen_col,
 Returns the credit level based on the formula `max(1.0 - (gen_row[gen_col] / c.benchmark), 0.0)`. 
 """
 function get_credit(c::CreditByBenchmark, data, gen_row::DataFrameRow)
-    gen_emis_rate = gen_row[c.gen_col]
-    credit = min.(1.0, max.( 1.0 .- gen_emis_rate ./ c.benchmark, 0.0))
+    if row_comparison(gen_row, parse_comparisons(c.gen_filters)) # meets gen filters
+        gen_emis_rate = gen_row[c.gen_col]
+        credit = min.(1.0, max.( 1.0 .- gen_emis_rate ./ c.benchmark, 0.0))
+    else
+        credit = 0
```

Comment:
I wonder if we should try to match the return type of `gen_emis_rate`?  I.e. a container, when appropriate.